_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp

```diff
@@ -1008,6 +1009,8 @@ mfxStatus VAAPIEncoder::CreateAuxilliaryDevice(
         m_caps.MaxNumOfROI = 0;
     }
 
+    m_caps.IntraRefreshBlockUnitSize = 2;
```

Comment:
so the default behavior is changed. is it expected?

Reply:
by default m_caps.IntraRefreshBlockUnitSize = 0 and it is wrong value because of whole frame is updated only half